_posts/2019-03-12-so-you-want-to-be-an-engineer.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: epic
+title: "So You Want to Be an Engineer"
+date: "2019-03-12"
+author: [matt_dole]
+categories: [artsy, beginners, culture, engineering, people, team]
+---
+
+First of all, that's very exciting! Software engineering is pretty darn cool—you get to learn lots of new things,
+understand the technology you use every day better, and contribute to the mysterious maw known as "the internet".
+
+Last February, I also decided that I wanted to pursue computer engineering. I'd been at Artsy for a bit less than
+two years at that point, first as a marketing intern working on SEO and then as a coordinator on the CRM (read:
+email) team. I'd consistently been working on small technical projects; first doing
+[some work](https://github.com/artsy/positron/commit/3176282a3ea94c626e9d851b7c0dd27a1bb0fcb4) on a tool for SEO
+optimization for our Editorial team, then building
+[emails with MJML](http://artsy.github.io/blog/2018/11/19/mjml/), and a few other bits and bobs. But I didn't think
+of it as a serious pursuit.
+
+Mostly, that was due to my experience programming in the past—I did about half a CS major in undergrad. At the
+time, I felt that programming wasn't right for me, and I dropped the major during my third year.
+
+It was Artsy's Engineering team that convinced me that programming was something that I both wanted to and could
+do. Our engineers have always welcomed learners and been happy to answer questions and empower other teams to do
+technical work. I eventually realized that the parts of my work where I was coding were the parts I enjoyed the
+most, and that I would likely feel more fulfilled if I made programming my full-time occupation.
+
+Here's what that journey looked like. Hopefully my experience proves helpful to you as you begin (or finish) yours!
+
```

Comment:
Awesome post!

Should we consider a `<!-- more -->` break here? Seems to be the norm for most other posts ([example](https://github.com/artsy/artsy.github.io/pull/542/files#diff-4f9b169eefd5e4a9385873b20df42c03R15)).

Reply:
I added this here: https://github.com/artsy/artsy.github.io/commit/fbc95c432fe4b2516b228f34a8cd667520713cf5

We should add a rule to the [Dangerfile](https://github.com/artsy/artsy.github.io/blob/source/Dangerfile) in this repo to require the string `"<!-- more -->"` to be included in all files in the `_posts/` directory. Would anyone like to pair?

Reply:
I think @mdole should have first pairing dibs, but I'd be interested as a backup!

Reply:
Haha thanks both! I'm certainly up for pairing on it, or we could do a mini-mob. Might be fun! 

Sent you both a cal invite for pairing on this Friday afternoon and gave you permission to modify if there's another time you'd prefer.

Reply:
Mob sounds great! Always happy to help more engineers write Danger rules 😄